app/javascript/application.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter} from 'react-router-dom'
+import { Provider } from 'react-redux';
+import store from './redux/store'
+import App from './components/App'
+
+ReactDOM.render(
+ ( 
+<BrowserRouter>
+      <Provider store={store}>
+        <App />
+      </Provider>
+    </BrowserRouter>
+ ),
+  document.getElementById('root'),
+  
+);
```

Comment:
- [ ] When I Run the React app uses the URL to displays a random greeting I got the following error
-  Error: Can’t resolve ‘react-router-dom’: This error is occurring because the `react-router-dom` module is not found in your project. You can install it by running the following command in your project’s root directory:
`npm install react-router-dom`

- Error: Can’t resolve ‘./redux/store’: This error suggests that the file store.js in the ./redux/ directory cannot be found. Make sure that the file exists and the path is correct. The path is relative to the file where you’re trying to import it.So kindly Fix the issues.
![image](https://github.com/DianaBeki/hello-rails-react/assets/93598189/da60e1f8-02bb-49a6-b641-bc40ad35e3f1)